### PR TITLE
attemp to fix HTML print

### LINKED ISSUE #18: Extra characters when printing in Pluto.jl
I was trying to using `Configurations.jl` in `Pluto.jl`. When making an option type object, the printing shows extract characters. Not sure if this is an issue more suitable here or in `Pluto.jl`..
extra_characters

### PR DESCRIPTION
**Pull request by Roger-luo**
close #18

### CODE REVIEW

File: src/printing.jl [hunk 1]
```diff
@@ -10,6 +10,13 @@ function show_option(io::IO, m::MIME"text/plain", x)
     indent_print(io, ")")
 end
 
+function show_option(io::IO, m::MIME"text/html", x)
+    buf = IOBuffer()
+    show(buf, MIME("text/plain"), x)
+    printer = HTMLPrinter(buf; root_class="configurations-option-type")
+    show(io, m, printer)
```
Comment (fonsp) [inline src/printing.jl:17]: you can add:
```julia
write(io, """
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/JuliaDocs/ANSIColoredPrinters.jl@0.0.1/docs/src/assets/default.css" />
""")
```

Comment (Roger-luo) [reply src/printing.jl:21]: ah thanks, now it works, but with an ugly `Main.workspace` prefix tho.

**Comment by Roger-luo** [comment]
still not perfect, but I guess I'll merge this first. Perhaps we can special case option types using the trait `is_option` inside Pluto after we swap out the original manual implementation to provide better visualizations.

**Comment by fonsp** [comment]
@Roger-luo How can I help? Do you want me to look into how you can get Pluto's default struct display method?

**Comment by Roger-luo** [comment]
@fonsp  yeah definitely! I currently only mimic the layout of Pluto so it works in other HTML environments (with the drop-down button). I'm not sure how should I do it for Pluto, perhaps we can overload a `show` for `MIME"application/pluto"`?

**Comment by fonsp** [comment]
I don't think it is possible right now... will revisit!

**Pull request merged**